java-advanced/task1/jmp-cloud-bank-impl/src/main/java/jmp/cloud/bank/BankImpl.java

```diff
@@ -0,0 +1,17 @@
+package jmp.cloud.bank;
+
+import jmp.bank.api.Bank;
+import jmp.dto.*;
+
+public class BankImpl implements Bank {
+    @Override
+    public BankCard createBankCard(User user, BankCardType type) {
+        switch (type) {
```

Comment:
Maybe can be simplify like `type.equals(CREDIT) ? new CreditBankCard(user) : new DebitBankCard(user)`